plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -1535,6 +1548,19 @@ public void testLimitPushdown()
                 .isNotFullyPushedDown(LimitNode.class);
     }
 
+    @Test
+    public void testPredicatePushdown()
+    {
+        assertThat(query("SELECT timestamp_col FROM " + ALL_TYPES_TABLE + " WHERE timestamp_col < TIMESTAMP '1971-01-01 00:00:00.000'"))
+                .isFullyPushedDown();
```

Comment:
The method name is `testPredicatePushdown` but the test is specific to timestamp type. Moving to `testTimestamp` seems better.